apps/dotcms-ui-e2e/src/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare namespace Cypress {
+    interface Chainable<Subject> {
+        form_request(
+            method: string,
+            url: string,
+            formData: FormData,
+            callback: Function
+        ): Chainable<any>;
+        iframe(callback?: Function): Chainable<any>;
+    }
+}
```

Comment:
What is this for?

Reply:
every time you define a `command` you need to specify the signature on the namespace Cypress, otherwise it complains